Libraries/TranscriptTestRunner/TestRunner.cs

```diff
@@ -98,10 +98,17 @@ public async Task<Activity> GetNextReplyAsync(CancellationToken cancellationToke
             }
         }
 
-        public async Task AssertReplyAsync(Action<Activity> validateAction, CancellationToken cancellationToken = default)
```

Comment:
Let's preserve this method so in manual tests they can inject their on validation code, something like:

```
            await runner.AssertReplyAsync(activity =>
            {
                Assert.Equal(ActivityTypes.Message, activity.Type);
                Assert.Equal("What skill would you like to call?", activity.Speak);
            })
```

Reply:
Done. Thanks!